docs/app/data-sharing.md

```diff
@@ -0,0 +1,219 @@
+
+### Assign an Existing Role
+
+1. Navigate to the **Share Options Menu**.
+2. Click the Role Input Box and a new side menu will appear _(shown above)_.
+3. Select the Role you'd like your Share to inherit permissions from.
+4. Click <span mi btn>check</span> to confirm.
+
+### Creating / Editing a Role
+
+_If your Role has the correct permissions configured_, you can create a new Role and assign it to a Share _"on the fly"_
+in the Share Options Menu. However, you will still need to navigate to **Settings > Roles and Permissions > [Role
+Name]** to configure any read permissions.
+
+![Creating / Editing a Role within Shares](https://cdn.directus.io/docs/v9/app-guide/content/data-sharing/data-sharing-20220217A/creating-and-edit-roles-menu-20220217A.webp)
+
+### Create a New Role
+
+1. Navigate to the **Share Options Menu**.
+2. Click <span mi icon>add</span> in the Role Input Box and the Role Options Menu will appear.
+3. Fill out the options to create the Role according to your needs.
+4. Click <span mi btn>check</span> to confirm Role creation.
+
+### Edit an Assigned Role
+
+1. Navigate to the **Share Options Menu**.
+2. Click <span mi icon>open_in_new</span> in the Role Input Box and the Role Options Menu will appear.
+3. Edit Role options according to your needs.
+4. Click <span mi btn>check</span> to confirm edits.
+
+::: warning You're creating / editing actual Roles!!
+
+If you create or edit a Role in this menu, these changes will take effect app-wide, not just in the context of the
+Share.
+
+:::
+
+## Limit Sharing by Collection
+
+You can limit which Collections a Role is allowed to share.
+
+1. Navigate to **Settings > Roles and Permissions**.
+2. Select the **[Role Name]** that will be responsible for creating the Share.
+3. Find the **[Collection]** you wish to set Filter(s) for and click the icon in the column under
+   <span mi icon>share</span> and a small menu will appear.
+4. Select <span mi>check</span> to enable and <span mi>block</span> to disable Shares on this collection.
+
+## Limit Sharing by Item
+
+Sometimes you may want a given Role to be able to share some Items but not others. Filters allow you to control exactly
+which Items a Role will be able to share, enabling you to do things like _only share reports that are marked as 'Done'_.
+
+1. Navigate to **Settings > Roles and Permissions**.
+2. Select the **[Role Name]** that will be responsible for creating the Share.
+3. Find the **[Collection]** you wish to set Filter(s) for and click the icon in the column under
+   <span mi icon>share</span> and a small menu will appear.
+4. Select <span mi icon>rule</span>**Use Custom** and a side menu will appear.
+5. Create your Filter(s) as needed.
+
+## Limit Sharing by Field
+
+As mentioned several times before, when you create a Share on some Item, you associate a Role _(and thus the Role's
```

Comment:
```suggestion
When you create a Share on some Item, you associate a Role _(and thus the Role's
```

Lets avoid "As mentioned several times before" type of language across the docs as it has a bit of a negative connotation. It reads to me like "Like we told you many times before you dummy, ..."